02.calendar/calendar.rb

```diff
@@ -0,0 +1,88 @@
+# コマンドライン引数・オプションの処理
+require 'optparse'
+opt = OptionParser.new
+
+
+days = {}
+
+opt.on('-y [VAL]') {|v| days[:y] = v }
+opt.on('-m [VAL]') {|v| days[:m] = v }
+
+opt.parse!(ARGV)
+
+# 今日を定義する
+require 'date'
+
+today = Date.today
+
+
+# 年月を定義
+day = today.day
+
+if (days[:y].to_i) == 0
+  year = today.year
+else year = (days[:y].to_i)
+end
```

Comment:
```suggestion
if (days[:y].to_i) == 0
  year = today.year
end
```

else でなにも処理していないので、条件式ごと削除しちゃいましょう